api/Window.json

```
@@ -1286,6 +1286,58 @@
           }
         }
       },
+      "deviceorientation": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/deviceorientation_event",
+          "description": "<code>deviceorientation</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "7"
+            },
+            "chrome_android": {
+              "version_added": true
```

We can add Chrome Android support for the first version released, as such:

```suggestion
              "version_added": "18"
```